cpp/src/barretenberg/honk/sumcheck/relations/grand_product_computation_relation.hpp

```diff
@@ -18,10 +18,11 @@ template <typename FF> class GrandProductComputationRelation {
      * This file handles the relation that confirms faithful calculation of the grand
      * product polynomial Z_perm. (Initialization relation Z_perm(0) = 1 is handled elsewhere).
      *
-     *  C(extended_edges(X)...) = ( z_perm(X) + lagrange_first(X) )*P(X) - ( z_perm_shift(X) + delta * lagrange_last(X) )*Q(X),
```

Comment:
Formatting hook changed this for an unknown reason, so I reformatted to something clear that the formatter would accept.